### PR TITLE
Add file headers including license and copyright information

### PR DESCRIPTION
**Pull request by csomh**
Signed-off-by: Hunor Csomortáni <csomh@redhat.com>

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Userspace Containerization team at Red Hat
```
Comment (lachmanfrantisek) [inline LICENSE:3]: Do we want to use `Userspace Containerization`?

Is the year correct?

File: dist2src/cli.py [hunk 2]
```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/python3
 
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: MIT
```
Comment (lachmanfrantisek) [inline dist2src/cli.py:5]: I remember some discussion about those shortcuts, that we can't use them. Maybe @TomasTomecek can remember?

Comment (csomh) [reply dist2src/cli.py:5]: According to [this training](https://trainingportal.linuxfoundation.org/learn/course/licensing-basics-for-developers-lfc191/file-notices/file-notices?page=7), this is a completely valid way to reference the license: it's unambiguous and explicit.

File: LICENSE (hunk 1 above)
Comment (csomh) [reply LICENSE]: Uh, that was a copy from `packit-service`. Fixed.

**Comment by csomh** [comment]
In the mean time I've learnt a few new things about this topic, I'll update this PR accordingly during this week.

**Comment by csomh** [comment]
Updated to remove year from the copyright notice and changed copyright holder to "Contributors to the Packit project".

PTAL

File: DCO [hunk 3]
```diff
@@ -0,0 +1,37 @@
+Developer Certificate of Origin
+Version 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
```
Comment (TomasTomecek) [inline DCO:4]: I'm assuming this means that LF developed the DCO and its their thing, hence the copyright; is that correct?

Comment (csomh) [reply DCO:4]: Yes, this is how I understand it.

**Pull request merged**